_redirects

```diff
@@ -0,0 +1,132 @@
+# Redirects from blog.fishtownanalytics.com -> getdbt.com/blog/ 
```

Comment:
A shame we can't use splat redirects for these! I assume the numbers at the end of the Ghost URLs prohibits that, yes?

Reply:
Yeah I assume that prevents us from using splat, not too familiar with those urls or what they used to be though 😬